beta/README.md

```diff
@@ -1,5 +1,121 @@
+- Create a PR containing the deployment example.
+  - Documentation and all configuration files should be contained in `./beta/<new deployment method name>/`
+  - In the PR, briefly describe the rationale for the proposed deployment (e.g., how it improves on existing methods, or how it addresses a use-case not already served).
+  - Describe anything that may still need to be implemented (e.g., Google Workspace support, certain vertical scaling, etc).
+  - Call out any specific tests or scenarios that you would like the reviewer to focus on. 
+- Update README.md at the root of the repository
+  - Add the deployment method to the list of beta deployments in [`README.md`](../README.md#beta-deployment) in the following format:   
+  `- ✨ **NEW** [Deployment Name](/beta/deployment-name)`
+- If you are a 1Password employee, assign reviewers from the Solutions Architect or AOP developer teams. 
+  - If you are a community member, a 1Password employee will review your PR when time and resources permit.
+- The beta deployment example will be merged into Main when:
+  - The reviewer(s) have tested the example successfully according to the use case it's designed to address.
+  - The documentation conforms to the criteria above. 
+  - All outstanding changes requested by reviewers have been addressed in a way that satisfies the reviewer. 
+  - The reviewer has, in their sole discretion, determined that the example is worth including in the repository, regardless of how well the example meets the criteria above. 
+
+> **Note**  
+> 1Password, in it's sole discretion, may choose to not publish examples for _any reason_, even if the example meets or exceeds all of the above criteria. 
+
+
+## Promoting a deployment example out of beta
+
+This section describes:
+
+- The criteria for promoting a deployment example out of "beta".
+- The process for graduating a deployment example from beta to stable.
+
+### Criteria for promoting a deployment example out of beta
+
+A deployment example may be in beta for whatever length of time is required to meet the promotion criteria. 
+
+- Functionality
+  - The deployment example _must_ work for all identity providers 1Password supports (unless otherwise exempt from that criteria). 
+  - The deployment example _must_ be capable of scaling to accommodate different provisioning needs (unless otherwise exempt from that criteria).
+  - The deployment example _must_ provide the complete automated user provisioning experience described by 1Password and in the documentation accompanying the deployment example. 
+  - Feedback from customers using the deployment example is nearly universally positive. Any critical feedback has been addressed to the greatest possible extent during the preceeding beta period. 
+  - The deployment example _must_ provide the complete automated user provisioning experience described by 1Password and in the documentation accompanying the deployment example. 
```

Comment:
duplicated bullet from above

Reply:
This may have been a quirk of doing this in VScode using a new GH plugin someone who shall remane nameless suggest I use ;)